tests/umock_c_with_lock_factory_int/minipal/umock_threadapi_pthread.c

```diff
@@ -0,0 +1,127 @@
+// Copyright (c) Microsoft. All rights reserved.
+// Licensed under the MIT license. See LICENSE file in the project root for full license information.
+
+#define _DEFAULT_SOURCE
```

Comment:
>_DEFAULT_SOURCE [](start = 8, length = 15)

what's this?

Reply:
[https://stackoverflow.com/questions/29201515/what-does-d-default-source-do](https://stackoverflow.com/questions/29201515/what-does-d-default-source-do)

---
In reply to: [530657003](https://github.com/Azure/umock-c/pull/177#discussion_r530657003) [](ancestors = 530657003)